assets/system/environment.yaml

```diff
@@ -0,0 +1,16 @@
+image:
+  name: azureml/curated/docker-tools
```

Comment:
so image build and docker-utils in one single env?

Reply:
correct